onnxruntime/core/framework/inlined_containers.h

```diff
@@ -20,21 +20,88 @@
 #endif
 
 namespace onnxruntime {
+/// Inspired by LLVM SmallVector with ONNX Runtime adjuments for abseil.
```

Comment:
is this sufficient for copyright purposes?

Reply:
Added Copyright on the top along with the derived/modified code notice per LLVM license requirements.